pkg/gateway/model_build_targetgroup.go

```diff
@@ -375,3 +396,25 @@ func (t *latticeServiceModelBuildTask) buildTargetGroupName(_ context.Context, b
 			t.route.Namespace())
 	}
 }
+
+func buildTargetGroupIpAddressType(_ context.Context, svc *corev1.Service) (string, error) {
```

Comment:
- naming, what we are building here? to looks like getIpAddrType from Service
- context not in use, please remove
- please add unit test